components/frontend/src/header_footer/Menubar.jsx

```diff
@@ -55,12 +44,14 @@ export function Menubar({
                     <Stack direction="row" spacing={2} sx={{ flexGrow: 1 }}>
                         <HomeButton
                             atReportsOverview={atReportsOverview}
-                            openReportsOverview={openReportsOverview}
-                            setSettingsPanelVisible={setSettingsPanelVisible}
+                            openReportsOverview={() => {
+                                setVisibleSettingsPanel("")
+                                openReportsOverview()
```

Comment:
Github is highlighting this line differently than the one before, is it expecting a semicolon to end the previous line or so?